### ISSUE TITLE
Access to embedded fields

### ISSUE THREAD

**Issue by tufitko** [antonmedv/expr#105]
found bug when tries to access to embedded fields

Example:
```
type Env struct{
	C
}

type C struct {
	A
	B
}

type A struct {
	Field string
}

type B struct {
	Field int
}

func main() {
	_, err := expr.Compile("C.A.Field == '' && C.B.Field == 0", expr.Env(Env{}), expr.AsBool())
	fmt.Println(err)
}
```
Output:
```
type main.C has no field A (1:3)
 | C.A.Field == '' && C.B.Field == 0
 | ..^
```

**Comment by antonmedv** [comment]
Try to solve it 😁

**Comment by antonmedv** [comment]
Fixed in 337130da884e8b5c18caecfe5e44ec11da815e3b
